core/commands/tour.go

```diff
@@ -50,7 +49,6 @@ func tourRunFunc(req cmds.Request, res cmds.Response) {
 	}
 
 	var w bytes.Buffer
```

Comment:
it would make me really happy if we could change this to `w := new(bytes.Buffer)`